source/material_model/visco_plastic.cc

```diff
@@ -252,7 +252,8 @@ namespace aspect
             out.reaction_terms[i][c] = 0.0;
 
           // Calculate changes in strain invariants and update the reaction terms
-          rheology->strain_rheology.fill_reaction_outputs(in, i, rheology->min_strain_rate, plastic_yielding, out);
+          if (in.requests_property(MaterialProperties::reaction_rates))
```

Comment:
As far as I understand this needs to be `MaterialProperties::reaction_terms`, and not `MaterialProperties::reaction_rates`. (see what happens inside StrainDependent<dim>::fill_reaction_outputs, it sets out.reaction_terms).